src/App.js

```diff
@@ -1,11 +1,31 @@
 import React from 'react'
+import { Switch, Route, Redirect } from 'react-router'
+import { Layout } from 'antd'
 
-import AppLayout from './core/components/layout'
+import { Content } from 'common/ui-kit'
+import { Header, Footer } from 'core/components'
+import { PortalPage, LoginPage, NotFoundPage } from 'pages/index'
 
-import './App.scss'
+import './App.css'
+
+const isLogged = true
 
 function App() {
-  return <AppLayout />
+  return (
+    <Layout>
+      <Header />
+      <Content>
+        <Switch>
+          <Route exact path="/login" component={LoginPage} />
+          <Route exact path="/404" component={NotFoundPage} />
+          <Route path="/*">
+            {isLogged ? <PortalPage /> : <Redirect to="/login" />}
```

Comment:
what's the logic behind this ? isLogged is true everytime

Reply:
isLogged is temporary variable, that help us imitate login behavior. In future we wiil take it from redux store

Reply:
so you can return only PortalPage by now